lib/opentok.js

```diff
@@ -521,6 +524,24 @@ OpenTok = function (apiKey, apiSecret, env) {
    *      as broadcast streams. If you include RTMP streaming, you can specify up to five
    *      target RTMP streams (or just one).
    *      </p>
+   *      <>
+   *      For HLS, include a single hls property in the outputs object. This object includes
+   *      the following optional properties:
+   *      <ul>
+   *       <li>
+   *        dvr (Boolean) — Whether to enable DVR functionality — rewinding, pausing, and
+   *        resuming — in players that support it (true), or not (false, the default). With
+   *        DVR enabled, the HLS URL will include a ?DVR query string appended to the end.
+   *       </li>
+   *       <li>
+   *        lowLatency (Boolean) — Whether to enable low-latency mode for the HLSstream.
+   *        Some HLS players do not support low-latency mode. This feature is incompatible with
+   *        DVR mode HLS broadcasts.
```

Comment:
Please add the links from the REST API docs (for "DVR functionality" and "low-latency mode"). Also put dvr and lowLatency (at the beginning of the bullets) in code tags.

Reply:
And add other code tags to match the formatting in the REST API docs.